app/controllers/ApprovalController.scala

```diff
@@ -40,10 +45,19 @@ class ApprovalController @Inject() (approvalService: ApprovalService, cc: Contro
 
   def get(id: String): Action[AnyContent] = Action.async { _ =>
     approvalService.getById(id).map {
-      case Right(process) => Ok(process)
+      case Right(approvalProcess) => Ok(Json.toJson(approvalProcess))
```

Comment:
if the service returns json as scratch and publish, rather than an ApprovalProcess object, it removes the need to propgate the implicit reads/writes beyond the service

Reply:
Can chat about this as not sure if you are saying to just return the process part of the ApprovalProcess (which is a JsObject) or return the whole ApprovalProcess but as json earlier in chain. I'm guessing after our discussions today, this method will be called as part of the 2i review/fact check so may just need the embedded json process.

Reply:
service return json similar to as scratch and publish 
def getById(id: String): Future[RequestOutcome[JsObject]] so it mirrors the existing save method on the service
def save(approvalProcess: JsObject): Future[RequestOutcome[String]]

Reply:
Changed method to return the embedded process to match scratch and published